### PR TITLE
Some additional tweaks and cleanup for v3.

### PR DESCRIPTION
**Pull request by pope1ni**

### CODE REVIEW

File: flightdataaccessor/formats/base.py [hunk 1]
```diff
@@ -101,6 +98,10 @@ def __len__(self):
         """Count of parameters in the flight data format object."""
         return len(self.keys())
 
+    def _ipython_key_completions_(self):
+        """Custom tab completions for __getitem__ in IPython >=5.0."""
+        return sorted(self.keys())
```
Comment (pope1ni) [inline flightdataaccessor/formats/base.py]: I think that this little bit could be very handy for us... :wink:

Comment (pope1ni) [reply flightdataaccessor/formats/base.py]: Should allow for typing something like ``fdf['Alt`` and then getting tab completion of the available parameter names...

Comment (kirku1g) [reply flightdataaccessor/formats/base.py]: Great idea. I had no idea implementing this was so straightforward :smiley:

**Comment by pope1ni** [comment]
> LGTM. Neatens the code up nicely.

Ta. I'll leave this for Marek to pull in when he comes back to working on this soon.

**Comment by pope1ni** [comment]
Did you incorporate any of these changes out of interest?

**Pull request closed without merging**